docker/shadow-hand/kinetic/Dockerfile

```diff
@@ -0,0 +1,55 @@
+FROM shadowrobot/build-tools:xenial-kinetic-ide
+
+ENV remote_shell_script="https://raw.githubusercontent.com/shadow-robot/sr-build-tools/master/ansible/deploy.sh"
+
+COPY additional_bashrc_cmds /tmp/additional_bashrc_cmds
```

Comment:
maybe it is a good idea to put this file into one place and download it using wget as one-liner to avoid duplication and if we will want to add something in the future we will change only one file :)